packages/plugin-input-exercise/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@edtr-io/plugin-input-exercise",
+  "version": "0.0.0",
+  "license": "MIT",
+  "author": "edtr.io Consortium",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "edtr-io/edtr-io",
+  "scripts": {},
+  "dependencies": {
+    "algebra.js": "^0.2.0",
+    "string": "^3.0.0"
+  },
+  "devDependencies": {
```

Comment:
For the future @metzgaria: Please review the dependencies before committing. In your code, both `@edtr-io/core` and `@edtr-io/ui` were missing. Furthermore, the `@types/*` packages were declared as dependency (instead of the preferred devDependency). :)